src/core/binding.c

```diff
@@ -998,7 +1003,12 @@ QuicBindingProcessStatelessOperation(
                 (uint8_t*)&Token,
                 (uint16_t)SendDatagram->Length,
                 SendDatagram->Buffer);
-        CXPLAT_DBG_ASSERT(SendDatagram->Length != 0);
+        if (SendDatagram->Length == 0) {
+            if (!CxPlatIsRandomMemoryFailureEnabled()) {
+                CXPLAT_DBG_ASSERT(FALSE);
+            }
```

Comment:
If we're going to keep the assert (which I'm unsure about my self) replace this with the following
```suggestion
            CXPLAT_DBG_ASSERT(CxPlatIsRandomMemoryFailureEnabled());
```